styles/base/utils.css

```diff
@@ -0,0 +1,48 @@
+/* Heading Tags */
+h1 {
+  font-size: 2.25rem;
+  line-height: 1.2;
+  font-weight: 600;
+}
+
+h2 {
+  font-size: 2rem;
+  line-height: 1.2;
+  font-weight: 600;
+}
+
+h3 {
+  font-size: 1.75rem;
+  line-height: 1.2;
+  font-weight: 600;
+}
+
+h4 {
+  font-size: 1.5rem;
+  line-height: 1.2;
+  font-weight: 600;
+}
+
+h5 {
+  font-size: 1.25rem;
+  line-height: 1.2;
+  font-weight: 600;
+}
+
+h6 {
+  font-size: 1rem;
+  line-height: 1.2;
+  font-weight: 600;
+}
```

Comment:
instead of giving same style repeatedly, you can do like this : 
h1,h2,h3,h4,h5,h6{
  line-height: 1.2;
  font-weight: 600;
}

Reply:
Yes..makes sense..will do that change